integration-tests/src/test/java/oracle/kubernetes/operator/ItJrfModelInImage.java

```diff
@@ -0,0 +1,243 @@
+         + BaseTest.getProjectRoot() 
+         + "/kubernetes/samples/scripts " 
+         + getResultDir(),
+         true),
+         "Failed: Copy sample dir to the result dir");
+    
+    //start DB and create RCU
+    dbNamespace = "db" + String.valueOf(getNewSuffixCount());
+    assertDoesNotThrow(() -> DbUtils.createNamespace(dbNamespace),
+        "Failed to create namespace: " + dbNamespace);
+    dbPort = 30011 + getNewSuffixCount();
+    dbUrl = "oracle-db." + dbNamespace + ".svc.cluster.local:1521/devpdb.k8s";
+    assertDoesNotThrow(() -> DbUtils.createDbRcu(getResultDir(), dbPort, dbUrl, 
+        rcuSchemaPrefix, dbNamespace));
+    LoggerHelper.getLocal().log(Level.INFO,"RCU schema is created for test: " 
+        + testClassName 
+        + " dbNamespace is: " + dbNamespace 
+        + " dbUrl:" + dbUrl 
+        + " dbPort: " + dbPort
+        + " rcuSchemaPrefix" + rcuSchemaPrefix); 
+        
+    // create operator
+    if (operator == null) {
+      Map<String, Object> operatorMap = createOperatorMap(getNewSuffixCount(),
+          true, testClassName);
+      operator = assertDoesNotThrow(() -> TestUtils.createOperator(operatorMap, RestCertType.SELF_SIGNED));
+      assertNotNull(operator);
+      LoggerHelper.getLocal().log(Level.INFO, "Operator is created for {0}", testClassName);
+      
+      domainNS = ((ArrayList<String>) operatorMap.get("domainNamespaces")).get(0);
+      namespaceList.append((String)operatorMap.get("namespace"));
+      namespaceList.append(" ").append(domainNS);
+    }
+  }
+  
+  /**
+   * This method will run once after all test methods are finished. It delete both RCU and DB pods
+   */
+  @AfterEach
+  public void unPrepare() {
+    LoggerHelper.getLocal().log(Level.INFO, "Is going to drop RCU schema and stop DB for {0}", testClassName);
+    assertDoesNotThrow(() -> DbUtils.deleteRcuPod(getResultDir()),
+        "Failed: drop RCU schema");
+    assertDoesNotThrow(() -> DbUtils.deleteDbPod(getResultDir()),
+        "Failed: stop DB");
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories.
+   */
+  @AfterAll
+  public static void staticUnPrepare() {
+    assertDoesNotThrow(() -> tearDown(new Object() {
+        }.getClass().getEnclosingClass().getSimpleName(), namespaceList.toString()),
+         "tearDown failed");
+  }
+
+  /**
+   * Create and deploy a JRF domain using model in image. Save and restore walletFileSecret. After shutting down 
+   * the domain, enable walletFileSecret in the domain yaml file. Restart the domain with the modified domain yaml 
+   * file that reuses the same RCU schema. Verify in the restarted domain the required pods, services are created 
+   * and the servers are ready.
+   */
+  @Test
+  public void testReuseRCU2Deployments() {
+    Assumptions.assumeTrue(QUICKTEST);
+    String testMethodName = new Object() {
+    }.getClass().getEnclosingMethod().getName();
+    assertDoesNotThrow(() -> logTestBegin(testMethodName));
```

Comment:
you don't need an assert on printing a log message...
asserts are for checking things that the test did to see if they worked as expected

Reply:
protected void logTestBegin(String testName) throws Exception {
    LoggerHelper.getLocal().log(Level.INFO,
        "+++++++++++++++++++++++++++++++++---------------------------------+");
    LoggerHelper.getLocal().log(Level.INFO, "BEGIN " + testName);
    // renew lease at the beginning for every test method, leaseId is set only for shared cluster
    TestUtils.renewK8sClusterLease(getProjectRoot(), getLeaseId());
  }
It does more than just print log message and it throws Exception.

Reply:
@maggiehe00 you can remove TestUtils.renewK8sClusterLease(getProjectRoot(), getLeaseId() call as we are not using lease concept anymore. You don't need assert here.

Reply:
OK. In my test class I will remove assert.

Reply:
Done